cafebabel/static/css/styles.css

```diff
@@ -2175,10 +2189,14 @@ a.section-link.creative {
   margin-bottom: 5rem;
 }
 
-.home article .article-detail h3 {
+article .article-detail h3 {
   font-size: 2.2rem;
 }
 
+article .article-detail h3 a:hover:after {
```

Comment:
Can you double check that `:hover:after` works? I would have written `:after:hover`

Reply:
It does work? I used it lines 643, 673, 851, 875, 923, 991, 1478, 1861, 2039, 2156 etc. ;-)